src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,62 @@
-import React from "react";
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
```

Comment:
We should use a consistent importing style in this repo. I already used named exports in other components, so from now on, please import MUI components like this instead of default exports

`import { Button}  from "@material-ui/core";`